### PR TITLE
Test/helpdesk locators

### PR DESCRIPTION
**Pull request by JaswiniP**
Added couple more id's in help desk.

### CODE REVIEW

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml [hunk 1]
```diff
@@ -24,21 +24,21 @@
             {
                 <div class="govuk-summary-list__row">
                     <dt id="edubase-trigger-1" class="govuk-summary-list__key">EduBase trigger 1</dt>
-                    <dd class="govuk-summary-list__value">@Model.Establishment.HelpdeskTrigger1</dd>
+                    <dd id="edubase-trigger" class="govuk-summary-list__value">@Model.Establishment.HelpdeskTrigger1</dd>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:27]: Does this need to be `edubase-trigger-1` as you added to one of the other ids yesterday or is it the only one so does not need numbering?

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml [hunk 2]
```diff
@@ -24,21 +24,21 @@
...
                 </div>
             }
             @if (Model.DisplayPolicy.HelpdeskPreviousLocalAuthorityId)
             {
                 <div class="govuk-summary-list__row">
                     <dt id="previous-local-authority" class="govuk-summary-list__key">Previous local authority</dt>
-                    <dd class="govuk-summary-list__value">@Model.HelpdeskPreviousLocalAuthorityName</dd>
+                    <dd id="previous-local-authority-info" class="govuk-summary-list__value">@Model.HelpdeskPreviousLocalAuthorityName</dd>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:34]: What is the `-info` part for? I'm assuming you're going to assert the text value in this element so probably worth swapping out `-info` for `-value` is that's the case.

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml [hunk 3]
```diff
@@ -24,21 +24,21 @@
...
                 </div>
             }
             @if (Model.DisplayPolicy.HelpdeskPreviousEstablishmentNumber)
             {
                 <div class="govuk-summary-list__row">
                     <dt id="previous-establishment-number" class="govuk-summary-list__key">Previous establishment number</dt>
-                    <dd class="govuk-summary-list__value">@Model.Establishment.HelpdeskPreviousEstablishmentNumber</dd>
+                    <dd id="previous-establishment-number-info" class="govuk-summary-list__value">@Model.Establishment.HelpdeskPreviousEstablishmentNumber</dd>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:41]: What is the `-info` part for? I'm assuming you're going to assert the text value in this element so probably worth swapping out `-info` for `-value` is that's the case.

**Review by ScottDawsonDfE — APPROVED** [review]

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml (hunk 1 above)
Comment (JaswiniP) [reply Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:27]: this is the value of the edubase trigger 1 to differentiate both I used edubase trigger

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml (hunk 2 above)
Comment (JaswiniP) [reply Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:34]: sure will update it now

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml (hunk 3 above)
Comment (JaswiniP) [reply Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:41]: will update and push the code

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml [hunk 4]
```diff
@@ -10,7 +10,7 @@
     <div class="detail-summary">
 
         <h2 id="helpdesk-notes" class="govuk-heading-s">Helpdesk notes:</h2>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:12]: Can we have this as `helpdesk-notes-key`.

File: Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml [hunk 5]
```diff
@@ -24,21 +24,21 @@
             {
                 <div class="govuk-summary-list__row">
                     <dt id="edubase-trigger-1" class="govuk-summary-list__key">EduBase trigger 1</dt>
```
Comment (antk43) [inline Web/Edubase.Web.UI/Areas/Establishments/Views/Establishment/Partials/TabHelpdesk.cshtml:26]: Can we have this as `edubase-trigger-1-key`.

**Review by antk43 — APPROVED** [review]

**Pull request merged**